yb-voyager/cmd/analyzeSchema.go

```diff
@@ -927,7 +927,7 @@ func generateHTMLReport(Report utils.SchemaReport) string {
 	//Broad details
 	htmlstring := "<html><body bgcolor='#EFEFEF'><h1>Database Migration Report</h1>"
 	htmlstring += "<table><tr><th>Database Name</th><td>" + Report.SchemaSummary.DBName + "</td></tr>"
-	htmlstring += "<tr><th>Schema Name</th><td>" + Report.SchemaSummary.SchemaName + "</td></tr>"
+	htmlstring += "<tr><th>Schema Name(s)</th><td>" + Report.SchemaSummary.SchemaNames + "</td></tr>"
```

Comment:
did you verify the HTML? Since the variable `SchemaNames` here is a list now, it might need to handled with the `range` clause. You can refer to the `assessment.template` file for that